CDK/Linux/hello-world/cdk.json

```diff
@@ -4,21 +4,20 @@
     "buildCompletionNotificationEmails": ["AlejandroRosalez@example.com"],
     "ImageBuilderPipelineConfigurations": [
       {
-          "name": "sampleimg",
-          "dir": "./image-builder-components",
-          "instanceProfileName": "ImageBuilderInstanceProfile",
-          "cfnImageRecipeName": "standalone-testrecipe02",
-          "version": "1.1.0",
-          "ssmParameterName":"ec2image_ami",
-          "parentImage": {
-              "ap-southeast-2": { "amiID": "ami-0b7dcd6e6fd797935" },
-              "ap-southeast-1": { "amiID": "ami-055d15d9cfddf7bd3" },
-              "us-east-1": { "amiID": "ami-04505e74c0741db8d" },
-              "us-east-2": { "amiID": "ami-0fb653ca2d3203ac1" },
-              "us-west-1": { "amiID": "ami-01f87c43e618bf8f0" },
-              "us-west-2": { "amiID": "ami-0892d3c7ee96c0bf7" }
-          }
+        "name": "imagebuilder-example",
+        "components": ["example-component"],
+        "instanceProfileName": "ImageBuilderInstanceProfile",
+        "cfnImageRecipeName": "imagebuilder-example-recipe",
+        "version": "1.0.0",
+        "parentImage": {
+          "ap-southeast-2": { "amiID": "ami-0b7dcd6e6fd797935" },
+          "ap-southeast-1": { "amiID": "ami-055d15d9cfddf7bd3" },
+          "us-east-1": { "amiID": "ami-04505e74c0741db8d" },
+          "us-east-2": { "amiID": "ami-0fb653ca2d3203ac1" },
+          "us-west-1": { "amiID": "ami-01f87c43e618bf8f0" },
+          "us-west-2": { "amiID": "ami-0892d3c7ee96c0bf7" }
```

Comment:
Are we able to update these to the latest Ubuntu 20.04 AMIs?

Eg, the one for `ap-southeast-2` was created on 30 November 2021, whereas the latest (`ami-02e8cbf7681c3ae51`) is from 10 August 2022.

---

An improvement here would be to list the EC2 Image Builder ARN for the parent image instead of the Image ID's directly, eg `arn:aws:imagebuilder:us-east-1:aws:image/ubuntu-server-20-lts-arm64/x.x.x` (for us-east-1)

Reply:
Those AMI IDs has been updated to the image builder ARNs for latest Ubuntu Server 20 LTS x86. The code can accommodate both image builder arn and ami ID in this field. Also the configuration in this file is only an example. When user use this sample to start build their AMI images, they should check and update those parent image IDs to make sure the base image is what they will need.